nextflow.config

```diff
@@ -143,19 +143,19 @@ process.shell = ['/bin/bash', '-euo', 'pipefail']
 def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file    = "${params.tracedir}/execution_timeline_${params.unique_id}.html"
+    file    = "${params.tracedir}/execution_timeline_${params.unique_id}_${trace_timestamp}.html"
```

Comment:
Could we separate this out so that it's changed in the CLIMB config to be as before? I like knowing exactly what the filename will be without having to do any fuzzy matching since I won't have the exact timestamp.

Reply:
Can do

Reply:
I think I did already actually! It's in the climb.config. Does that not load first? The reason I don't want the params.unique_id in by default is that it creates a '[:]' directory for these outputs if you forget to create a unique_id (or don't want to)

Reply:
You changed the tracedir but not the location of the file itself I think? Unless I'm missing it

Reply:
Ah, good spot. I'll update.